tests/config/junos/OSPF_NEIGHBOR_DOWN/inactive_timer/yang.json

```diff
@@ -0,0 +1,62 @@
+{
+  "yang_message": {
+    "network-instances": {
+      "network-instance": {
+        "global": {
+          "protocols": {
+            "protocol": {
+              "ospf": {
+                "ospfv2": {
+                  "areas": {
+                    "area": {
+                      "0.0.0.0": {
+                        "interfaces": {
+                          "interface": {
+                            "ge-0/0/0.0": {
+                              "neighbors": {
+                                "neighbor": {
+                                  "1.1.1.1": {
+                                    "state": {
+                                      "adjacency-state-change-reason-message": "dead timer",
```

Comment:
I'm wondering how diverse is this message and if we could define a typeref as with the others, e.g., `DEAD_TIMER` (so it will be standard across other platforms). We can leave this as an open discussion for later, but it would be nice if you could build up a list of the messages you've seen.

Reply:
This was just me being lazy with the test and I should have fixed it in the last commit. The reason message is tough is because there are many different messages that relate to the `INACTIVE_TIMER` event type. For example, whether of not BFD is in play. 

This is the same problem in the BGP state changes. As you can see there are already a number mapping required to cover all state transitions in the BGP state machine. IMO it would be very unwieldy to then throw in all the human event messages because there are cases where several messages map to one state change. 

I think it would be more feasible down the road if we could define these messages separate from the log mappings. These would allow us to map several different vender messages for an event to some model value while not having to keep up with tens/hundreds of log mappings for a single log type.

Reply:
Good points. My opinion is that we should try to map them as much as possible (if they are not completely random and there's a decent amount of mappings needed).

> I think it would be more feasible down the road if we could define these messages separate from the log mappings.

Good suggestion - after this is merged, I will open an issues so we can discuss further and see in time what's the best way to approach this, after we'll have more examples and feedback from the users / developers.

Reply:
Let's discuss this further and build up a list of possible messages: https://github.com/napalm-automation/napalm-logs/issues/176